MantidQt/CustomInterfaces/inc/MantidQtCustomInterfaces/Reflectometry/ReflSaveTabPresenter.h

```diff
@@ -64,6 +64,9 @@ class MANTIDQT_CUSTOMINTERFACES_DLL ReflSaveTabPresenter
   /// Obtains all available workspace names
   std::vector<std::string> getAvailableWorkspaceNames();
 
+  /// Predicate to checks if a workspace is of a group or table workspace type
+  struct isGroupOrTable;
```

Comment:
Consider using the check in `operator()` as a lambda:
```
      [](const std::string &wsName) {
        // Predicate to checks if a workspace is either of group or table types
        // or not
        return AnalysisDataService::Instance().retrieveWS<WorkspaceGroup>(
                   wsName) ||
               AnalysisDataService::Instance().retrieveWS<ITableWorkspace>(
                   wsName);
      }
```

it will save a struct declaration, and the `isGroupOrTable` isn't used anywhere else